site/docs/config.toml

```diff
@@ -46,7 +46,9 @@ ignoreFiles = [
   ".*\\.yml$",
   ".*\\.clang-format$",
   "/__pycache__/.*",
+  "/bazel-.*/.*",
   "/build/.*",
+  "/build-.*/.*",
```

Comment:
Which directories is this meant to cover? Presumably it adds `build-bin` and `build-out` to the ignore list. (That might well be fine but I'm curious why we didn't need it before if those are the only `build-*` directories being excluded.)

Reply:
Yeah, that's correct.

I think it manages to document those directories without errors and that to generate the real documentation it doesn't generate them before making the documentation so the effect of this should just be to make saner more representative documentation generation on the workstations.